### PR TITLE
Improve db seed step for quick start example

### PR DESCRIPTION
**Pull request by skalinets**
Instead of using UI let's use simple scripts to create the table and seed it with data.

### CODE REVIEW

File: create-table.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+docker run --tty \
```
Comment (jeqo) [inline create-table.sh]: We could use docker-compose exec to make this command simpler:

```bash
docker-compose exec postgres psql -U postgres -c 'CREATE TABLE IF NOT EXISTS source_table ( id SERIAL PRIMARY KEY, name VARCHAR(100) NOT NULL);'
```

Also, could be moved to `Makefile`

File: insert-data.sh [hunk 2]
```diff
@@ -0,0 +1,8 @@
+docker run --tty \
```
Comment (jeqo) [inline insert-data.sh]: Same here:

```
docker-compose exec postgres psql -U postgres -c 'INSERT INTO source_table ( name ) VALUES (MD5(random()::text));'
```

> for some reason github suggestions are not allowed here.

File: README.md [hunk 3]
```diff
@@ -57,25 +57,32 @@ docker-compose up -d
 ```
 
 Steps to test:
-1. Navigate to http://localhost:8080 and login using `postgres` as server, `postgres` as username and `example` as password
+* Create a table `source_table`:
 
-2. Create a table `source_table` with an auto-increment `id` and `name` field
+```bash
+./create-table.sh
```
Comment (jeqo) [inline README.md]: Call `make` task instead once commands are moved to makefile.

**Review by jeqo — COMMENTED** [review]
thanks for adding this @skalinets 

It'll need to be rebased first to get latest. Left some small change recommendations before merging.

**Comment by jeqo** [comment]
@skalinets I've fill the last gaps to get this ready to merge. Thanks!

**Comment by skalinets** [comment]
great, thank you!

**Pull request merged**